src/Telemetry/Library/Telemetry.csproj

```diff
@@ -12,7 +12,7 @@
       <_Parameter1>Microsoft.Quantum.Telemetry.Tests$(PublicKeySuffix)</_Parameter1>
     </AssemblyAttribute>
   </ItemGroup>
-  <ItemGroup>
-    <PackageReference Include="Microsoft.Applications.Events.Client" Version="1.1.1.337" />
+  <ItemGroup>

```

Comment:
It's odd that this gets highlighted as a diff when the lines appear identical... could this be a whitespace change?

Reply:
This is very odd. When excluding whitespace changes, this goes away, but even showing whitespaces I don't see any difference. I'll see if I can make the diff go away.